CommandScripts/autonomy.py

```diff
@@ -171,7 +204,8 @@ def start_mission(self):
             if homing_end in response:
                 while True:
                     self.current_GPS = self.GPS_data.get_position(f"{self.url}/gps")
-                    command = self.get_steering(self.current_GPS[0], self.current_GPS[1], self.GPS_target[0], self.GPS_target[1])
+                    # command = self.get_steering(self.current_GPS, self.GPS_target)
+                    command = self.get_ctl_steering(self.current_GPS, self.GPS_target)
                     response += self.serial.read_serial()
                     self.get_rover_status()
                     if response != "No data received":
```

Comment:
We will need to edit this here since the new serial class I wrote has a better function for reading and writing.